src/main/java/picard/fingerprint/FingerprintChecker.java

```diff
@@ -116,6 +113,10 @@ public File getReferenceFasta() {
 
     private boolean allowDuplicateReads = false;
     private double pLossofHet = 0;
+    private int locusMaxReads = 0;
+    private String defaultSampleID = "<UNKNOWN>";
+
+    private Set<Path> missingRGFiles = new HashSet<>();
```

Comment:
It looks like this can be `final`
```suggestion
    private final Set<Path> missingRGFiles = new HashSet<>();
```